docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -361,100 +310,110 @@ You should be able to move the custom styles to the root class key.
-- [Modal] event.defaultPrevented is now ignored.
-  The new logic closes the Modal even if `event.preventDefault()` is called on the key down escape event.
-  `event.preventDefault()` is meant to stop default behaviors like clicking a checkbox to check it, hitting a button to submit a form, and hitting left arrow to move the cursor in a text input etc.
-  Only special HTML elements have these default behaviors.
-  People should use `event.stopPropagation()` if they don't want to trigger a `onClose` event on the modal.
+  ```css
+  box-sizing: border-box;
+  ```
+  It solves issues with the `fullWidth` prop.
+- [InputBase] Remove the `inputType` class from `InputBase`.
 
-### Portal
+### Tooltip
+
+- [Tooltip] The child needs to be able to hold a ref. The [composition guide](/guides/composition/#caveat-with-refs) explains
+  the migration strategy.
+- [Tooltip] Appears only after focus-visible focus instead of any focus.
 
-- [Portal] The child needs to be able to hold a ref when `disablePortal` is used.
+### Typography
+
+- [Typography] Remove the deprecated typography variants. You can upgrade by performing the following replacements:
+  - display4 => h1
+  - display3 => h2
+  - display2 => h3
+  - display1 => h4
+  - headline => h5
+  - title => h6
+  - subheading => subtitle1
+  - body2 => body1
+  - body1 (default) => body2 (default)
+- [Typography] Remove the opinionated `display: block` default typography style.
+  You can use the new `display?: 'initial' | 'inline' | 'block';` property.
+- [Typography] Rename the `headlineMapping` property to `variantMapping` to better align with its purpose.
 
   ```diff
-  class Component extends React.Component {
-    render() {
-      return <div />
-    }
-  }
-  -const MyComponent = props => <div {...props} />
-  +const MyComponent = React.forwardRef((props, ref) => <div ref={ref} {...props} />);
-  <Portal><Component /></Portal>
-  <Portal><MyComponent /></Portal>
-  <Portal><div /></Portal>
+  -<Typography headlineMapping={headlineMapping}>
+  +<Typography variantMapping={variantMapping}>
   ```
+- [Typography] Change the default variant from `body2` to `body1`.
+  A font size of 16px is a better default than 14px.
+  Bootstrap, material.io or even our documentation use 16px as a default font size.
+  14px like Ant Design is understandable as Chinese users have a different alphabet.
+  We document 12px as the default font size for Japanese.
+- [Typography] Remove the default color from the typography variants.
+  The color should inherit most of the time. It's the default behavior of the web.
+- [Typography] Rename `color="default"` to `color="initial"` following the logic of #13028.
+  The usage of *default* should be avoided, it lakes semantic.
 
-### Slide
+### Node
 
-- [Slide] The child needs to be able to hold a ref.
+- [Drop official node 6 support](https://github.com/nodejs/Release/blob/eb91c94681ea968a69bf4a4fe85c656ed44263b3/README.md#release-schedule), you should upgrade to node 8.
```

Comment:
What is "*official* node 6"?

Reply:
Meaning: "Drop official support of node 6"

Reply:
So, it's unofficially supported?

Reply:
I guess 😁. Realistically I don't think the addition of 'official' adds any meaning. I would get rid of it, anything we don't support can still be used if someone wanted to use it.

Reply:
Agreed. *Official* maybe has some other translations in French?